CovidApp/API/Services/ApiManager.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import UIKit
+
+enum ApiType {
+
+    case getTotal
+
+    var baseUrl: String {
+        return "https://api.covid19api.com/"
+    }
+
+    var headers: [String: String] {
+        switch self {
+        default:
+            return [:]
+        }
+    }
+
+    var path: String {
+        switch self {
+        case .getTotal:
+            return "summary"
+        }
+    }
+
+    var request: URLRequest {
+        let url = URL(string: path, relativeTo: URL(string: baseUrl)!)!
+        var request = URLRequest(url: url)
+        request.allHTTPHeaderFields = headers
+        switch self {
+        default:
+            request.httpMethod = "GET"
+            return request
+        }
+    }
+}
+
+class ApiManager {
+
+    static let shared = ApiManager()
```

Comment:
Do you mean classic singleton or singleton plus? If the former, then declare a `private init`.